src/sentry/api/serializers/models/userreport.py

```diff
@@ -55,20 +55,24 @@ def __init__(self, environment_func=None):
     def get_attrs(self, item_list, user):
         from sentry.api.serializers import GroupSerializer
 
-        # TODO(dcramer); assert on relations
-        groups = {
-            d["id"]: d
-            for d in serialize(
-                set(i.group for i in item_list if i.group_id),
-                user,
-                GroupSerializer(environment_func=self.environment_func),
-            )
-        }
+        groups = list(Group.objects.filter(id__in=[i.group_id for i in item_list if i.group_id]))
```

Comment:
Can you send in a `set()` here to reduce the raw number of ids sent into the query, or does Django do that automatically? In the end it doesn't matter to the database, but may still make the query smaller.